Sources/ApodiniDelta/Change/ChangeType.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  File.swift
+//  
+//
+//  Created by Eldi Cano on 27.03.21.
+//
+
+import Foundation
+
+/// Distinct change types that might occur
+enum ChangeType: String, Value {
```

Comment:
It only serves as a helper type to distinguish different changes currently. I will need it in the future to filter certain change types, and since some Change objects are generics, having this enum avoids using a type erasure.